site/data/dynamo.ts

```diff
@@ -0,0 +1,30 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, QueryCommand } from "@aws-sdk/lib-dynamodb";
+import { ptSituationElementSchema } from "@create-disruptions-data/shared-ts/siriTypes.zod";
+import logger from "../utils/logger";
+
+export const getDdbClient = (): DynamoDBClient => new DynamoDBClient({ region: "eu-west-2" });
+
+export const getDdbDocumentClient = (ddbClient = getDdbClient()): DynamoDBDocumentClient =>
+    DynamoDBDocumentClient.from(ddbClient);
+
+export const getDisruptionsDataFromDynamo = async (ddbDocClient: DynamoDBDocumentClient, tableName: string) => {
```

Comment:
On second thoughts it might be better if we just create an instance of the doc client outside of this function but in this file, then we wouldn't need to pass it through

Reply:
Will update now